src/components/Views/List/List.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Button from '../../UI/Button/Button';
+import styles from './List.scss';
+
+class List extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      cities: [
+        {
+          id: 1,
+          name: 'Minsk',
+          temperature: '+ 43',
+        },
+        {
+          id: 2,
+          name: 'Moscow',
+          temperature: '+ 21',
+        },
+        {
+          id: 3,
+          name: 'Bratislava',
+          temperature: '+ 15',
+        },
+      ],
+    };
+  }
+
+  render() {
+    const { cities } = this.state;
```

Comment:
if **cities** var is not changed in time - then there is not reason to put in instate variable. It could be event taken out of the component's scope.

Reply:
It's demand by ESlinter :)